lib/wanda/messaging/mapper.ex

```diff
@@ -84,6 +94,10 @@ defmodule Wanda.Messaging.Mapper do
     }
   end
 
+  defp map_target(%Target{agent_id: agent_id, checks: target_checks}) do
```

Comment:
Strange the conversion from `checks` to `target_checks` hehe
But I guess is a copy/paste from somewhere else

Reply:
👀  you're right